### PR TITLE
support serverless remove

### LINKED ISSUE #4: support serverless remove
serverless remove should delete all the backup tables and global table connection. I'm working on the patch now using the
```
      'remove:remove': function () {
        deleteGlobalAndBackupTables(serverless)
      },
```

hook. PS love the plugin, there is really limited support for deploying global tables without cloudformation.

### PR DESCRIPTION
**Pull request by johndww**
goes through all backup region tables and deletes them if they exist. fixes #4

### CODE REVIEW

**Comment by rrahul963** [comment]
@johndww - do you mind addressing the review comments? once resolved i can merge your PR

**Comment by johndww** [comment]
@rrahul963 I see no review comments.

File: README.md [hunk 1]
```diff
@@ -17,17 +17,20 @@ plugins:
...
+
 ```yaml
 custom:
   globalTables:
   - regions: # list of regions in which you want to set up global tables
       - region-1
       - region-2
-    tableKey: 'TABLE_KEY' # Cloudformation output key name if the table is created as part of same serverless service
-    tableName: 'TABLE_NAME' # if table is not part of the service then specify the table name. If tableKey param exists then tableName is ignored.
+    tableName: 'TABLE_NAME' # *RECOMMENDED* if table is not part of the service then specify the table name. If tableKey param exists then tableName is ignored.
```
Comment (rrahul963) [inline README.md]: why are you saying that it's RECOMMENDED not to create the table as part of service? @johndww

File: index.js [hunk 2]
```diff
@@ -156,6 +193,36 @@ const createGlobalDynamodbTable = async function createGlobalDynamodbTable(serve
...
+    serverless.cli.consoleLog(`DeleteGlobalTable: ${chalk.yellow('Starting to delete backup region global tables...')}`)
+    const awsCredentials = serverless.getProvider('aws').getCredentials()
+
+    const globalTablesOptions = serverless.service.custom.globalTables
+    if (!globalTablesOptions || globalTablesOptions.length === 0) {
+      return
+    }
+    await Promise.all(globalTablesOptions.map((option) => {
+      if (option.tableKey) {
+        serverless.cli.consoleLog(`DeleteGlobalTable: ${chalk.yellow('Warning: backup region global tables not removed. ' +
```
Comment (rrahul963) [inline index.js]: @johndww why cant we delete the global table if original table is part of the service?

**Comment by rrahul963** [comment]
@johndww - with the latest update to the package this change is no longer valid. 
Please feel free to update the code or i'll do it as and when i get a chance.

**Pull request closed without merging**